spec/policies.yml

```diff
@@ -13,34 +13,43 @@ components:
             id: db-password
             owner: !host db-host
 
+    LoadPolicy:
+      type: object
+      properties:
+        created_roles:
+          type: object
+        version:
+          type: number
+      example:
+        {
+          "created_roles": {
+            "myorg:host:database/db-host": {
+              "id": "myorg:host:database/db-host",
+              "api_key": "309yzpa1n5kp932waxw6d37x4hew2x8ve8w11m8xn92acfy672m929en"
+            }
+          },
+          "version": 1
+        }
+
   responses:
-    LoadedPolicy:
+    LoadPolicy:
```

Comment:
I changed the schema name to `LoadedPolicy` I still think the response should be named `LoadPolicy` because we have been naming the response objects after the operationID (or one of the operationID's) the response belongs to.

Reply:
So with the perspective of a second review, I can understand this convention organizationally. Most paths are pointing to responses of the same name, and most responses point to more specific schemas, which works for me.